pkg/get/tools.go

```diff
@@ -505,5 +505,30 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/v{{.Version}}/pack-v{{
 		https://github.com/{{.Owner}}/{{.Repo}}/releases/download/v{{.Version}}/{{.Name}}-v{{.Version}}.{{$osStr}}-{{$archStr}}{{$extStr}}`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:   "open-policy-agent",
+			Repo:    "opa",
+			Name:    "opa",
+			Version: "0.24.0",
+			URLTemplate: `
```

Comment:
We can now switch to binary templates, so that the version can be left empty and found at runtime.

The last line should be: `v{{.Version}}/opa_{{$osStr}}_amd64{{$extStr}}` if switching to the BinaryTemplate as only the final part is required.